govcd/api_vcd_test.go

```diff
@@ -747,6 +747,28 @@ func (vcd *TestVCD) getAdminOrgAndVdcFromCleanupEntity(entity CleanupEntity) (or
 	return org, vdc, nil
 }
 
+func (vcd *TestVCD) removeLeftoverEntitiesByTestName(check *C, testName string) {
```

Comment:
Very similar to `removeLeftoverEntities`, except it filters cleanup entities by `testName`